package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "cordova-common": "^3.1.0",
+    "execa": "^2.0.4",
```

Comment:
Any reason to use an old version of execa? (see https://github.com/sindresorhus/execa/releases/tag/v3.2.0)

Reply:
Good catch